setup.py

```diff
@@ -63,6 +63,12 @@ def main():
     log_rotate_dir = '/etc/logrotate.d'
     log_rotate_files = ['scripts/apel-client']
 
+    # Python 3.x dependencies
+    _install_requires = ['mysqlclient', 'iso8601', 'python-ldap', 'dirq', 'future']
+    # Python 2.x dependencies
+    if sys.version_info <= (3,):
```

Comment:
```suggestion
    if sys.version_info < (3,):
```
Remember that you can't just flip the sign as the "3" is only inclusive on one side of the range.

Reply:
Resolved now.